cni/cmd/istio-cni-repair/main.go

```diff
@@ -18,8 +18,12 @@ package main
 
 import (
 	"fmt"
+	"net/http"
 	"os"
 
+	"github.com/prometheus/client_golang/prometheus"
```

Comment:
I think we have standardized on using OpenCensus (which wraps around prometheus).

See security/pkg/nodeagent/cache/monitoring.go for example, specifically the "istio.io/pkg/monitoring" package

Reply:
Done.